src/java.base/windows/native/libnio/ch/FileChannelImpl.c

```diff
@@ -36,8 +36,6 @@
 #include <Mswsock.h>
 #pragma comment(lib, "Mswsock.lib")
 
-static jfieldID chan_fd; /* id for jobject 'fd' in java.io.FileChannel */
-
 /**************************************************************
  * static method to store field ID's in initializers
  * and retrieve the allocation granularity
```

Comment:
Should we remove the "to store field ID's in initializers" part from this comment?

Reply:
Removed in 0c707e41492a682967f522ce03894f87a133a16f.